### ISSUE TITLE
New submissions for Mon,  6 Nov 23

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#140]
# New submissions for Mon,  6 Nov 23
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', 'AGN']

#### A Direct Comparison between the use of Double Gray and Multiwavelength  Radiative Transfer in a General Circulation Model with and without  Radiatively Active Clouds
 - **Authors:** Isaac Malsky, Emily Rauscher, Michael T. Roman, Elspeth K. H. Lee, Hayley Beltz, Arjun Savel, Eliza M. R. Kempton, L. Cinque
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP)
 - **Arxiv link:** https://arxiv.org/abs/2311.01506
 - **Pdf link:** https://arxiv.org/pdf/2311.01506
 - **Abstract**
 Inhomogeneous cloud formation and wavelength-dependent phenomena are expected to shape hot Jupiter atmospheres. We present a General Circulation Model (GCM) with multiwavelength "picket fence" radiative transfer and radiatively active, temperature dependent clouds, and compare the results to a double gray routine. The double gray method inherently fails to model polychromatic effects in hot Jupiter atmospheres, while picket fence captures these non-gray aspects and performs well compared to fully wavelength-dependent methods. We compare both methods with radiatively active clouds and cloud-free models, assessing the limitations of the double gray method. Although there are broad similarities, the picket fence models have larger day-night side temperature differences, non-isothermal upper atmospheres, and multiwavelength effects in the presence of radiatively active clouds. We model the well-known hot Jupiters HD 189733 b and HD 209458 b. For the hotter HD 209458 b, the picket fence method prevents clouds from thermostating dayside temperatures, resulting in hotter upper atmospheres and the dissipation of dayside clouds. Differences in the temperature structures are then associated with nuanced differences in the circulation patterns and clouds. Models of the cooler HD 189733 b have global cloud coverage, regardless of radiative transfer scheme, whereas there are larger differences in the models of HD 209458 b, particularly in the extent of the partial cloud coverage on its dayside. This results in minor changes to the thermal and reflected light phase curves of HD 189733 b, but more significant differences for the picket fence and double gray versions of HD 209458 b.
#### Tentative detection of cyanoformamide NCCONH2 in space
 - **Authors:** Juan Li, Donghui Quan, Junzhi Wang, Xia Zhang, Xing Lu, Qian Gou, Feng Gao, Yajun Wu, Edwin Bergin, Shanghuo Li, Zhiqiang Shen, Fujun Du, Meng Li, Siqi Zheng, Xingwu Zheng
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2311.01721
 - **Pdf link:** https://arxiv.org/pdf/2311.01721
 - **Abstract**
 The peptide-like molecules, cyanoformamide (NCCONH2), is the cyano (CN) derivative of formamide (NH2CHO). It is known to play a role in the synthesis of nucleic acid precursors under prebiotic conditions. In this paper, we present a tentative detection of NCCONH2 in the interstellar medium (ISM) with the Atacama Large Millimeter/submillimeter Array (ALMA) archive data. Ten unblended lines of NCCONH2 were seen around 3sigma noise levels toward Sagittarius B2(N1E), a position that is slightly offset from the continuum peak. The column density of NCCONH2 was estimated to be 2.4\times 10^15 cm ^-2, and the fractional abundance of NCCONH2 toward Sgr B2(N1E) was 6.9\times10^-10. The abundance ratio between NCCONH2 and NH2CHO is estimated to be ~0.01. We also searched for other peptide-like molecules toward Sgr B2(N1E). The abundances of NH2CHO, CH3NCO and CH3NHCHO toward Sgr B2(N1E) were about one tenth of those toward Sgr B2(N1S), while the abundances of CH3CONH2 was only one twentieth of that toward Sgr B2(N1S).
#### ProPane: Image Warping with Fire
 - **Authors:** A. S. G. Robotham, R. Tobar, S. Bellstedt, S. Casura, R. H. W. Cook, J. C. J. D'Silva, L. J. Davies, S. P. Driver, J. Li, L. P. Garate-Nuñez
 - **Subjects:** Instrumentation and Methods for Astrophysics (astro-ph.IM); Cosmology and Nongalactic Astrophysics (astro-ph.CO); Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2311.01761
 - **Pdf link:** https://arxiv.org/pdf/2311.01761
 - **Abstract**
 In this paper we introduce the software package ProPane, written for the R data analysis language. ProPane combines the full range of wcslib projections with the C++ image manipulation routines provided by the CImg library. ProPane offers routines for image warping and combining (including stacking), and various related tasks such as image alignment tweaking and pixel masking. It can stack an effectively unlimited number of target frames using multiple parallel cores, and offers threading for many lower level routines. It has been used for a number of current and upcoming large surveys, and we present a range of its capabilities and features. ProPane is already available under a permissive open-source LGPL-3 license at github.com/asgr/ProPane (DOI: 10.5281/zenodo.10057053).
#### The metallicity variations along the chromosome maps: The Globular  Cluster 47 Tucanae
 - **Authors:** A. F. Marino, A. P. Milone, E. Dondoglio, A. Renzini, G. Cordoni, H. Jerjen, A. I. Karakas, E. P. Lagioia, M. V. Legnardi, M. McKenzie, A. Mohandasan, M. Tailo, D. Yong, T. Ziliotto
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2311.01871
 - **Pdf link:** https://arxiv.org/pdf/2311.01871
 - **Abstract**
 The "chromosome maps" (ChMs) of globular clusters (GCs) have revealed that these ancient structures are not homogeneous in metallicity in various ways, and in different natures. The Type II GCs generally display larger variations, sometimes coupled with slow neutron capture (s) element enrichment on the ChMs redder sequences, which has been interpreted as due to multiple generations of stars. On the other hand, most GCs have inhomogeneous first populations (1P) in the form of large ranges in the Delta(F275W,F814W) values, pointing towards a not fully mixed pristine molecular cloud. We analyse the chemical composition the GC 47 Tucanae, which shows both inhomogeneous 1P stars and, although not formally a Type II GC, hosts a small number of stars distributed on a red side of the main stream of ChM stars. Our results suggest that 1P stars are not homogeneous in the overall metallicity, with variations of the order of ~0.10 dex in all the chemical species. The anomalous stars distributed on a redder sequence of the ChM, are further enriched in metals, but without any evidence for a significant enrichment in the s elements. Our three second population stars located on the normal component of the map, have metallicities similar to those of the metal-richer 1P group, suggesting that this population formed from these stars. Although three stars is a too-small sample to draw strong conclusions, the low spread in metals of these objects might point towards a formation in a fully mixed medium, possibly after a cooling flow phase.
#### Neural ODEs as a discovery tool to characterize the structure of the hot  galactic wind of M82
 - **Authors:** Dustin Nguyen, Yuan-Sen Ting, Todd A. Thompson, Sebastian Lopez, Laura A. Lopez
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA); Fluid Dynamics (physics.flu-dyn)
 - **Arxiv link:** https://arxiv.org/abs/2311.02057
 - **Pdf link:** https://arxiv.org/pdf/2311.02057
 - **Abstract**
 Dynamic astrophysical phenomena are predominantly described by differential equations, yet our understanding of these systems is constrained by our incomplete grasp of non-linear physics and scarcity of comprehensive datasets. As such, advancing techniques in solving non-linear inverse problems becomes pivotal to addressing numerous outstanding questions in the field. In particular, modeling hot galactic winds is difficult because of unknown structure for various physical terms, and the lack of \textit{any} kinematic observational data. Additionally, the flow equations contain singularities that lead to numerical instability, making parameter sweeps non-trivial. We leverage differentiable programming, which enables neural networks to be embedded as individual terms within the governing coupled ordinary differential equations (ODEs), and show that this method can adeptly learn hidden physics. We robustly discern the structure of a mass-loading function which captures the physical effects of cloud destruction and entrainment into the hot superwind. Within a supervised learning framework, we formulate our loss function anchored on the astrophysical entropy ($K \propto P/\rho^{5/3}$). Our results demonstrate the efficacy of this approach, even in the absence of kinematic data $v$. We then apply these models to real Chandra X-Ray observations of starburst galaxy M82, providing the first systematic description of mass-loading within the superwind. This work further highlights neural ODEs as a useful discovery tool with mechanistic interpretability in non-linear inverse problems. We make our code public at this GitHub repository (https://github.com/dustindnguyen/2023_NeurIPS_NeuralODEs_M82).
#### Physical properties of the eclipsing binary KIC 9851944 and analysis of  its tidally-perturbed p- and g-mode pulsations
 - **Authors:** Zachary Jennings, John Southworth, Kresimir Pavlovski, Timothy Van Reeth
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2311.02064
 - **Pdf link:** https://arxiv.org/pdf/2311.02064
 - **Abstract**
 Stars that are both pulsating and eclipsing offer an important opportunity to better understand many of the physical phenomena that occur in stars, because it is possible to measure the pulsation frequencies of stars for which the masses and radii are known precisely and accurately. KIC 9851944 is a double-lined detached eclipsing binary containing two F-stars which show both pressure and gravity mode pulsations. We present an analysis of new high-resolution spectroscopy of the system and high quality light curves from the Kepler and TESS space missions. We determine the masses and effective temperatures of the stars to 0.6% precision, and their radii to 1.0% and 1.5% precision. The secondary component is cooler, but larger and more massive than the primary so is more evolved; both lie inside the {\delta} Scuti and {\gamma} Doradus instability strips. We measure a total of 133 significant pulsation frequencies in the light curve, including 14 multiplets that each contain between 3 and 19 frequencies. We find evidence for tidal perturbations to some of the p- and g-modes, attribute a subset of the frequencies to either the primary or secondary star, and measure a buoyancy radius and near-core rotational frequency for the primary component. KIC 9851944 is mildly metal-rich and MIST isochrones from the MESA evolutionary code agree well with the observed properties of the system for an age of 1.25 Gyr.

by olozhika (Xing Yuchen). 

2023-11-06
